cmake/CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ option(onnxruntime_BUILD_OBJC "Build Objective-C library" OFF)
 option(onnxruntime_USE_PREINSTALLED_EIGEN "Use pre-installed EIGEN. Need to provide eigen_SOURCE_PATH if turn this on." OFF)
 option(onnxruntime_BUILD_BENCHMARKS "Build ONNXRuntime micro-benchmarks" OFF)
 
-option(onnxruntime_USE_TVM "Build TVM for code generation." OFF)
+option(onnxruntime_NUPHAR_USE_TVM "Build TVM for code generation." OFF)
```

Comment:
Why `onnxruntime_NUPHAR_USE_TVM` and not `onnxruntime_USE_NUPHAR_TVM`?

Reply:
@xadupre it does not metter for me it separated 'USE_TVM' after 'USE_STVM' renaming and 'USE_TVM' used by Nuphar. if you wish I can redo it

Reply:
Every other similar constant starts with `onnxruntime_USE...`. It seems better to follow the same pattern.

Reply:
Ok. done